components/cards/cards.css

```diff
@@ -0,0 +1,112 @@
+.main-card{
+    max-width: 300px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    position: relative;
+    border-radius: 0.5rem;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    box-sizing: border-box;
+    margin: 1.2rem;
+}
+
+.card-badge{
+    position: absolute;
+    top: 1.2rem;
+    background-color:#1FABD7;
+    color: white;
+    padding: 0rem 1rem;
+}
+
+.card-dismiss{
+    position: absolute;
+    top: 5px;
+    right: 10px;
+    color: white;
+    font-size: 1.2rem;
+}
+
+.card-dismiss:hover{
+    cursor: pointer;
+}
+
+.img-comp{
+    width: 100%;
+    border-radius: 0.5rem 0.5rem 0 0;
+}
+
+
+
+.info-div:hover{
```

Comment:
try using generic class names, something that shows what are you using it for.
for eg: card-info